src/app/Fake.Core.Trace/TraceListener.fs

```diff
@@ -110,7 +110,7 @@ type TagStatus =
 
 /// Defines Tracing information for TraceListeners
 [<RequireQualifiedAccess>]
-type TraceData = 
+type TraceData =
     | ImportData of typ:ImportData * path:string
     | BuildNumber of text:string
     | ImportantMessage of text:string
```

Comment:
Can you add a comment to `OpenTag` that `description` might be null in case it is missing (which is probably the best we can do for now, ideally we would change to `string option` but I don't want to make that breaking change here.